.circleci/scripts/build_image.sh

```diff
@@ -21,7 +21,7 @@ log 2 "Application Name:  ${APPNAME}"
 # Builds using top-level directory ($CURRENT_DIR/..) as context
 DOCKER_BUILDKIT=1 docker build -f app.Dockerfile -t $DOCKER_TAG_1 -t $DOCKER_TAG_2 -t $DOCKER_TAG_HEROKU --target $DOCKER_TARGET $CURRENT_DIR/../..
 
-if [ "$CIRCLE_BRANCH" == "master" ] || [ "$CIRCLE_BRANCH" == "production" ]; then
+if [ "$CIRCLE_BRANCH" == "master" ] || [ "$CIRCLE_BRANCH" == "production" ] || [ "$CIRCLE_BRANCH" == "3396-dummy-data" ]; then
```

Comment:
the plan is to remove this once we get into staging, we don't need extra dockerhub tags for this issue once it's merged in